src/model/Birch/birch_default_parameters.py

```diff
@@ -3,35 +3,31 @@
 import joblib
 from datetime import datetime
 
-# Hyperparameters
-n_clusters = 20
 
 # Initialize BIRCH model
-birch = Birch(n_clusters=n_clusters)
+birch = Birch()
 
 model = dict({
     'model': birch,
     'metadata': {
         'name': 'BIRCH',
-        'abbreviation': 'birch',
+        'abbreviation': 'BI',
```

Comment:
Tror ikke dette er vanlig??

Reply:
Nei, kanskje jeg skal legge til et felt. For jeg brukte dette i visualiseringen. Trengte korte navn